src/store/modules/actions/wallet.js

```diff
@@ -51,18 +53,22 @@ export function disconnectWallet({ state, commit }) {
   commit(types.WALLET_SET_LIKERINFO, null);
 }
 
-export async function restoreSession({ dispatch }) {
+export async function restoreSession({ dispatch, commit }) {
   const connector = await dispatch('initConnector');
   const session = connector.restoreSession();
   if (session) {
     await dispatch('initWallet', session);
   }
 }
 
-export async function initIfNecessary({ dispatch }) {
+export async function initIfNecessary({ dispatch, commit }) {
   const connector = await dispatch('initConnector');
   const session = await connector.initIfNecessary();
   if (session) {
     await dispatch('initWallet', session);
   }
+  const connectedWalletType = JSON.parse(
+    window.localStorage?.getItem(KEY_CONNECTED_WALLET_TYPE)
+  );
+  commit(types.WALLET_SET_METHOD_TYPE, connectedWalletType.method);
```

Comment:
You can get the connection method of the session by `session.method`

I suggest to put `WALLET_SET_METHOD_TYPE` to initWallet
```
  commit(types.WALLET_SET_METHOD_TYPE, session.method);
```